src/App.js

```diff
@@ -3,38 +3,69 @@ import styled from 'styled-components'
 import { RouterProvider } from 'react-router-dom'
 
 import { router } from './utils/Router'
-import { Grid } from '@digicatapult/ui-component-library'
+import { Grid, SidePanel } from '@digicatapult/ui-component-library'
 import { Context } from './utils/Context'
 
-const colorMap = {
-  emma: '#AAED93',
-  heidi: '#FDB6D4',
-  reginald: '#FCF281',
-}
+const personas = [
+  {
+    id: 'conor',
+    name: 'Conor Conor',
+    title: 'Conor Conor',
+    subtitle: 'The Hydrogen Consumer',
+    background: '#FDB6D4',
+  },
+  {
+    id: 'emma',
+    name: 'Emma Emma',
+    title: 'Emma Emma',
+    subtitle: 'The Energy Producer',
+    background: '#AAED93',
+  },
+  {
+    id: 'heidi',
+    name: 'Heidi Heidi',
+    title: 'Heidi Heidi',
+    subtitle: 'The Hydrogen Producer',
+    company: "Heidi's Hydroelectric Hydrogen",
+    // background: '#FDB6D4',
+    background: '#9EDCFA',
+  },
+  {
+    id: 'reginald',
+    name: 'Reginald Reginald',
+    title: 'Reginald Reginald',
+    subtitle: 'The Regulator',
+    background: '#FCF281',
+  },
+]
 
 const FullScreenGrid = styled(Grid)`
   height: 100lvh;
   width: 100lvw;
+  transition-delay: 0.3s;
   overflow: hidden;
+  box-sizing: content-box;
+  transition-property: margin-left border;
```

Comment:
instead of transitioning border (which is not animatable) perhaps we could transform padding? That way you can set the background color. Otherwise maybe tranform margin and use a before psuedo element to do the background. You also need to adjust the `width` with a calc and animate that?

Reply:
Do agree with your suggestion. Let me give it a go with a new SidePanel